src/transformers/trainer.py

```diff
@@ -1303,6 +1312,9 @@ def predict(self, test_dataset: Dataset, ignore_keys: Optional[List[str]] = None
             ignore_keys (:obj:`Lst[str]`, `optional`):
                 A list of keys in the output of your model (if it is a dictionary) that should be ignored when
                 gathering predictions.
+            metric_key_prefix (:obj:`str`, `optional`, defaults to "eval"):
```

Comment:
```suggestion
            metric_key_prefix (:obj:`str`, `optional`, defaults to :obj:`"eval"`):
```

Reply:
Why `eval` an object?

Reply:
I think we wrap all strings of the first line in the docs into a :obj:` ` . I was comparing to those lines e.g.: https://github.com/huggingface/transformers/blob/df3f4d2aef70d409b4d9bea18d8821693bea3877/src/transformers/models/bert/configuration_bert.py#L75